test/src/test/java/org/corfudb/integration/CorfuStoreIT.java

```diff
@@ -480,6 +488,115 @@ public void alterTableUsingDropTest() throws Exception {
         assertThat(shutdownCorfuServer(corfuServer)).isTrue();
     }
 
+    /**
+     * This test verifies that syncing a table with entries different from the schema it was
+     * opened with does not throw an error if the ProtobufSerializer is made aware of the
+     * different schema.
+     * @throws Exception
+     */
+    @Test
+    public void syncTableWithOldSchemaTest() throws Exception {
+        Process corfuServer = runSinglePersistentServer(corfuSingleNodeHost, corfuStringNodePort);
+
+        // PHASE 1 - Start a Corfu runtime & a CorfuStore instance
+        runtime = createRuntime(singleNodeEndpoint);
+
+        // Creating Corfu Store using a connected corfu client.
+        CorfuStore corfuStore = new CorfuStore(runtime);
+
+        // Define a namespace for the table.
+        final String someNamespace = "some-namespace";
+        // Define table name.
+        final String tableName = "EventInfo";
+
+        // Create & Register the table.
+        // This is required to initialize the table for the current corfu client.
+        Table<Uuid, SampleSchema.EventInfo, ManagedResources> table = corfuStore.openTable(
+                someNamespace,
+                tableName,
+                Uuid.class,
+                SampleSchema.EventInfo.class,
+                ManagedResources.class,
+                // TableOptions includes option to choose - Memory/Disk based corfu table.
+                TableOptions.builder().build());
+
+        Uuid key = Uuid.newBuilder().setLsb(0L).setMsb(0L).build();
+        SampleSchema.EventInfo value = SampleSchema.EventInfo.newBuilder().setName("simpleValue").build();
```

Comment:
looks like this record need not be constructed.

Reply:
done, thanks